addrmgr/addrmanager.go

```diff
@@ -74,6 +74,13 @@ type localAddress struct {
 	score AddressPriority
 }
 
+// LocalAddr describe network address information for a local address.
+type LocalAddr struct {
+	Address string `json:"address"`
```

Comment:
Shouldn't have json tags specified for a type in addrmgr.  They're only needed for types in the RPC types module.